app/src/main/java/de/westnordost/streetcomplete/ktx/Collections.kt

```diff
@@ -73,3 +73,6 @@ inline fun <T> Iterable<T>.sumByFloat(selector: (T) -> Float): Float {
 
 fun <T> Collection<T>.containsExactlyInAnyOrder(other: Collection<T>): Boolean =
     other.size == size && containsAll(other)
+
+/** If null, returns an empty array. Otherwise, returns an array with the value as the only element */
+inline fun <reified T> T?.asSingleArray(): Array<T> = if (this == null) arrayOf() else arrayOf(this)
```

Comment:
Hm. There is a reason why `getTitleArgs` returns an array and not a `String?` - because there may be several args.

I would very much prefer a `arrayOfNotNull` function instead. There is already a `listOfNotNull` function, so this would be consistent.